bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -113,7 +113,8 @@ PreProcessor::PreProcessor(shared_ptr<MsgsCommunicator> &msgsCommunicator,
                            metricsComponent_.RegisterCounter("preProcConsensusNotReached"),
                            metricsComponent_.RegisterCounter("preProcessRequestTimedout"),
                            metricsComponent_.RegisterCounter("preProcReqSentForFurtherProcessing"),
-                           metricsComponent_.RegisterCounter("preProcPossiblePrimaryFaultDetected")},
+                           metricsComponent_.RegisterCounter("preProcPossiblePrimaryFaultDetected"),
+                           metricsComponent_.RegisterGauge("currentPreProcReqNum", 0)},
```

Comment:
All other metrics have "preProc" prefix. Please rename it to preProcInFlyRequestsNum.

Reply:
done